tests/Validator/PasswordValidatorTest.php

```diff
@@ -0,0 +1,45 @@
+<?php
+
+namespace App\Tests\Validator;
+
+use App\Validator\Password;
+use App\Validator\PasswordValidator;
+use Symfony\Component\Validator\Test\ConstraintValidatorTestCase;
+
+class PasswordValidatorTest extends ConstraintValidatorTestCase
+{
+    public function testPasswordIsNull(): void
+    {
+        $this->validator->validate(null, new Password());
+        $this->buildViolation('Password not valid!')->assertRaised();
+    }
+
+    /**
+     * @dataProvider providePasswordValues
+     */
+    public function testPassword(string $password, bool $expected): void
+    {
+        $this->validator->validate($password, new Password());
+        if ($expected) {
+            $this->assertNoViolation();
+        } else {
+            $this->buildViolation('Password not valid!')->assertRaised();
+        }
+    }
+
+    public function providePasswordValues(): array
+    {
+        return[
+            ['', false, ['test']],
+            ['parola', false, ['test']],
+            ['parola!!!!!', false, ['test']],
+            ['Parola11!@', true, ['test']],
+            ['Parol a12!', false, ['test']],
+        ];
+    }
+
+    protected function createValidator(): PasswordValidator
+    {
+        return new PasswordValidator();
+    }
+}
```

Comment:
PSR-12 - new line at and of each file

Reply:
I did, but i guess github deleted the new line..